types/js-md5/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for js-md5 0.4
+// Type definitions for js-md5 0.7
```

Comment:
you're missing 'base64' export, if you intend to bump to .7. Otherwise the change is just niceties.

Reply:
Thanks. I updated tests https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63603/files#diff-0ea5d1a799ce76d975de29a1a13b3699d83183391934c8e35818397a7f96a916R9